application/src/main/java/eu/europa/ec/fisheries/uvms/activity/rest/resources/ReportDocumentResource.java

```diff
@@ -45,23 +45,9 @@ This file is part of the Integrated Fisheries Data Management (IFDM) Suite. The
     @Produces(value = {MediaType.APPLICATION_XML})
     @Consumes(value = {MediaType.APPLICATION_XML})
     public Object getReport(FLUXFAQueryMessage fluxfaQueryMessage) throws JAXBException {
-        DateTimeType startDate = fluxfaQueryMessage.getFAQuery().getSpecifiedDelimitedPeriod().getStartDateTime();
-        DateTimeType endDate = fluxfaQueryMessage.getFAQuery().getSpecifiedDelimitedPeriod().getEndDateTime();
-        LocalDateTime localStartDate = LocalDateTime.of(
-                startDate.getDateTime().getYear(),
-                startDate.getDateTime().getMonth(),
-                startDate.getDateTime().getDay(),
-                startDate.getDateTime().getHour(),
-                startDate.getDateTime().getMinute(),
-                startDate.getDateTime().getSecond());
-        LocalDateTime localEndDate = LocalDateTime.of(
-                endDate.getDateTime().getYear(),
-                endDate.getDateTime().getMonth(),
-                endDate.getDateTime().getDay(),
-                endDate.getDateTime().getHour(),
-                endDate.getDateTime().getMinute(),
-                endDate.getDateTime().getSecond());
-        FLUXFAReportMessage reportsByCriteria = faQueryService.getReportsByCriteria(localStartDate.toInstant(ZoneOffset.UTC), localEndDate.toInstant(ZoneOffset.UTC));
+        Instant startInstant = fluxfaQueryMessage.getFAQuery().getSpecifiedDelimitedPeriod().getStartDateTime().getDateTime().toGregorianCalendar().getTime().toInstant();
```

Comment:
Prettier but not pretty, but what can you do :)
I'm a bit worried though that the UTC ZoneOffset disappeared.

Reply:
Don't think that is it a problem in this case.